generic_config_updater/gcu_field_operation_validators.conf.json

```diff
@@ -19,7 +19,7 @@
             "mellanox_asics": {
                 "spc1": [ "ACS-MSN2700", "ACS-MSN2740", "ACS-MSN2100", "ACS-MSN2410", "ACS-MSN2010", "Mellanox-SN2700", "Mellanox-SN2700-D48C8" ],
                 "spc2": [ "ACS-MSN3800", "Mellanox-SN3800-D112C8" ],
-                "spc3": [ "ACS-MSN4700", "ACS-MSN4600", "ACS-MSN4600C", "ACS-MSN4410", "Mellanox-SN4600C-D112C8", "Mellanox-SN4600C-C64" ]
+                "spc3": [ "ACS-MSN4700", "ACS-MSN4600", "ACS-MSN4600C", "ACS-MSN4410", "Mellanox-SN4600C-D112C8", "Mellanox-SN4600C-C64", "Mellanox-SN4700-O8C48" ]
```

Comment:
need to handle the version support as well. currently spc3 -> 202205. we have 4600 for which we need support from 202205 but 4700 only from 202305

Reply:
discussed offline. platforms will always be onboarded with the right version. so this change should suffice